Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1

```diff
@@ -755,12 +764,22 @@ function Remove-VMHostStaticIScsiTargets {
     }
 
     $iSCSIAddressList = $iSCSIAddress.Split(",")
-
     $DatastoreDisks = Get-Datastore | Select-Object -ExpandProperty ExtensionData | Select-Object -ExpandProperty Info | Select-Object -ExpandProperty Vmfs | Select-Object -ExpandProperty Extent
     $TargetsChanged = $False
 
+    $VMHosts = $null
+    if ($VMHostName) {
+        $VMhosts = $Cluster| Get-VMHost -Name $VMHostName
```

Comment:
It looks like this should be `$VMHosts` as well (also lines 775 and 777)

Reply:
Fixed